uwblueprint/headsupguys
# Add MD builder short answer

## Notion ticket link

[[Module Builder] Short Answer](https://www.notion.so/uwblueprintexecs/Module-Builder-Short-Answer-1a1a3f4784d84ea29f40e78a1952a720)



## Implementation description
* Made Short Answer Module Builder Section



## Steps to test
1. Add short answer module builder section
2. Play around with values and ensure no new bugs are discovered
3. Look through the changed files on github and ensure no major coding qualms are present




## What should reviewers focus on?
* Module builder short answer section

![image](https://user-images.githubusercontent.com/24593519/150654799-5d236c0f-c44f-4013-9838-39a653b4d708.png)


## Checklist
